jbpm-recommendation-pmml-logistic-regression/pom.xml

```diff
@@ -0,0 +1,109 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jbpm</groupId>
+    <artifactId>jbpm</artifactId>
+    <version>7.27.0-SNAPSHOT</version>
```

Comment:
this caused the CI failure, please update to 7.29.0-SNAPSHOT as while merging it needs to be for new modules in line with master branch version

Reply:
@mareknovotny thanks! I'll update the PR.